File: benchmarks/src/main/resources/python/fibonacci_iterative.py
def fibonacci(n):
  if n <= 1:
    return n
  nMinusOne = 1.0
  nMinusTwo = 0.0
  for index in range(2, n + 1):
    nMinusOne = nMinusOne + nMinusTwo
    nMinusTwo = nMinusOne - nMinusTwo

  return nMinusOne

File: benchmarks/src/main/resources/js/fibonacci.iterative.js
function fibonacci(n) {
  if (n <= 1) {
    return n;
  }

  var nMinusOne = 1;
  var nMinusTwo = 0;
  for (var index = 2; index <= n; ++index) {
    nMinusOne += nMinusTwo;
    nMinusTwo = nMinusOne - nMinusTwo;
  }

  return nMinusOne;
}